src/twisted/conch/test/test_session.py

```diff
@@ -112,6 +116,23 @@ def getPty(self, terminal, window, modes):
             raise RuntimeError('not getting a pty')
 
 
+    def setEnv(self, name, value):
+        """
+        If the requested environment variable is 'LD_PRELOAD', fail.  If it
+        is 'IGNORED', raise ValueError, which should cause it to be silently
+        ignored.  Otherwise, store the requested environment variable.
+
+        (Real applications should normally implement an allowed list rather
+        than a blocked list.)
+        """
+        if name == b'LD_PRELOAD':
```

Comment:
This seems like a *suspiciously* specific value to choose for a stub test :).  Would you mind making it a less "set off security scanners" type value? :)